client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -144,9 +144,7 @@ export class DomainDetailsForm extends PureComponent {
 	}
 
 	getSubmitButtonText() {
-		return this.hasAnotherStep()
-			? this.props.translate( 'Continue' )
-			: this.props.translate( 'Continue to Checkout' );
+		return this.props.translate( 'Continue' );
 	}
```

Comment:
Since there's no logical operation here, we could simply remove the function and add it to `renderSubmitButton()` inside the [FormButton component](https://github.com/Automattic/wp-calypso/pull/32569/files#diff-7232cf58d72f88f6651b6db5cd6f3f54R156).

E.g.,

```
	renderSubmitButton() {
		return (
			<FormButton
				className="checkout__domain-details-form-submit-button"
				onClick={ this.handleSubmitButtonClick }
			>
				{ this.props.translate( 'Continue' ) }
			</FormButton>
		);
	}
```

Reply:
Great idea as well! I did not notice that, and I have updated now with 2c24df80a6605490151b393322a5f5450d7d3081.